jest/setup-enzyme.js

```diff
@@ -0,0 +1,3 @@
+const { configure } = require('enzyme');
```

Comment:
This was auto-generated so I figured may as well use Enzyme even if it's overkill for one test.